documentation/javadoc/pom.xml

```diff
@@ -41,7 +41,7 @@
             we import all types from dependant modules -->
         <dependency>
             <groupId>org.hibernate.ogm</groupId>
-            <artifactId>hibernate-ogm-infinispan</artifactId>
+            <artifactId>hibernate-ogm-infinispan-embedded</artifactId>
```

Comment:
I think you also need to change the `sourcepath` attribute a little further. And probably add the infinispan-remote module too.

Reply:
Good catch!